### PR TITLE
Metrics alerts

### PR DESCRIPTION
**Pull request by MaksTkachuk**

### CODE REVIEW

File: api_helper.py [hunk 1]
```diff
@@ -41,8 +47,14 @@ def get_query_data(query, slug, dt_from, dt_to, interval):
...
         gql_query = '{' + query + '(' + query_args_str + ')' + query_fields_str + '}'
-        response = execute_gql(gql_query)
-        return response[query]
+        attempts = 0
+        while attempts < 3:
+            try:
+                response = execute_gql(gql_query)
+                return response[query]
+            except SanError:
+                attempts += 1
```
Comment (valo) [inline api_helper.py]: Why do we need to make these attempts? Generally the API should be always working.

File: queries.py [hunk 2]
```diff
@@ -395,5 +395,9 @@
     "fundsRaisedEthIcoEndPrice",
     "fundsRaisedBtcIcoEndPrice",
     "tokenAgeConsumed",
-    "ethSpent"
+    "ethSpent",
+    "socialGainersLosersStatus",
+    "exchangeWallets",
+    "allExchanges",
+    "percentOfTokenSupplyOnExchanges"
```
Comment (valo) [inline queries.py]: The query for the total supply on exchange is currently this one https://api.santiment.net/graphiql?query=%7B%0A%20%20topHoldersPercentOfTotalSupply(slug%3A%22santiment%22%2C%20from%3A%222020-01-01T00%3A00%3A00Z%22%2C%20to%3A%222020-02-01T00%3A00%3A00Z%22%2C%20numberOfHolders%3A%2050)%7B%0A%20%20%20%20datetime%0A%20%20%20%20inExchanges%0A%20%20%20%20outsideExchanges%0A%20%20%20%20inTopHoldersTotal%0A%20%20%7D%0A%7D

File: api_helper.py (hunk 1 above)
Comment (MaksTkachuk) [reply api_helper.py]: Ivan told me that occasional errors can happen, it's not a rare thing, and I don't want tests to fail on each such occasion.

File: queries.py (hunk 2 above)
Comment (MaksTkachuk) [reply queries.py]: I use the correct one, the one in line 402 is ignored

**Pull request merged**